runtime/bindings-generator.cs

```diff
@@ -3522,6 +3547,16 @@ public static class Types {
 				IsX86Stret = true,
 				IsX64Stret = true,
 			};
+
+			public static TypeData MPSAxisAlignedBoundingBox = new TypeData {
+				ManagedType = "MPSAxisAlignedBoundingBox",
+				NativeType = "MPSAxisAlignedBoundingBox",
+				NativeWrapperType = "struct MPSAxisAlignedBoundingBoxWrapper",
+				RequireMarshal = true,
+				IsX86Stret = true,
+				IsX64Stret = true,
+				IsARMStret = true,
```

Comment:
unlikely to matter much but it's 64bits only so ARM and X86 stret should not be needed